### PR TITLE
Clarified trial clusters configuration information

### PR DESCRIPTION
**Pull request by zrk02**
Clarified trial clusters configuration to align with current requirements.
Reducing the potential for misreading the intentions.

### CODE REVIEW

File: content/en/docs/private-kubernetes/overview.md [hunk 1]
```diff
@@ -51,9 +51,7 @@ supported by Elastx Kubernetes CaaS.
 
 ### Trial clusters
 
-Trial clusters can be ordered in whatever configuration you wish. Do note
-however that trial cluster that do not meet at least the standard
-configuration will have to be rebuilt to be turned in to production ready clusters.
+Trial clusters follow the standard configuration. The only distinct difference is the time limited discount in price.
```
Comment (testviking) [inline content/en/docs/private-kubernetes/overview.md:54]: > The only distinct difference is the time limited discount in price.

Suggested edit:

"The word Trial refers to the initial trial-period and nothing else. After the trial-period, the same cluster will continue running and become billed via a monthly subscription."

Comment (testviking) [reply content/en/docs/private-kubernetes/overview.md:54]: Or maybe just: The word Trial refers to the initial trial-period and nothing else.

is enough.

Comment (testviking) [reply content/en/docs/private-kubernetes/overview.md:54]: It seems unfortunate to me that we need to call a cluster a Trial cluster. It is a cluster, and the trial-period is part of the sales process. What if we just remove the whole Trial clusters text?

Comment (zrk02) [reply content/en/docs/private-kubernetes/overview.md]: Updated according to the last suggestion.

**Review by testviking — APPROVED** [review]
Looks good.

**Pull request merged**